fake_spectra/rate_network.py

```diff
@@ -333,19 +302,82 @@ def GammaeHep(self,temp):
         """Collisional ionization rate for HeI in cm^3/s. Temp in K. Voronov 97, Table 1."""
         return self._Voronov96Fit(temp, 54.4, 1, 0.205e-08, 0.265, 0.25)
 
+
+class RecombRatesBadnell(object):
```

Comment:
Make it inherit from RecombRatesVerner96 - that way you don't need to repeat the functions that didn't change, like alphad, Gammae and _Voronov96Fit They just appear by the magic of inheritance